docs/api.md

```diff
@@ -52,6 +52,32 @@ coupling that inhibits encapsulation & reuse, often making testing difficult.
 
 Override to determine whether the ViewComponent should render.
 
+### #render_in(view_context, &block)
```

Comment:
These docs are meant to be auto-generated from YARD comments. Perhaps we could update the contributing guidelines / add a comment to the top of this file pointing folks in that direction?

Reply:
I'm not sure what you mean. In this PR the docs are generated from YARD comments. I ran `bundle exec rake docs:build` before committing, as per https://viewcomponent.org/CONTRIBUTING.html#documentation